tests/comparisons/Test/testBakeBehaviorTest.php

```diff
@@ -11,6 +11,13 @@ class ExampleBehaviorTest extends TestCase
 {
 
     /**
+     * Test subject
+     *
+     * @var \App\Model\Behavior\ExampleBehavior
+     */
+    public $Example = null;
```

Comment:
Is the = null necessary? This implies that its nullable, and the doc block would need |null attached.
It might be cleaner to omit it.